### DISCUSSION TITLE
/dev/iac

### DISCUSSION THREAD

**Discussion by rf23** [ish-app/ish#1316] [General]
has /de/ica in pull request https://github.com/ish-app/ish/pull/315 been integrated? 

I have been a very happy camper with ish on iPad and using Emacs for my daily work including read/reply emali in Emacs. One feature (the only feature) I really miss is to allow my email client in Emacs (VM) to launch an external app to view HTML mail and image or PDF attachments. Right now I had to put in a kludge to save attachments in Files app and manually open them after opening Files app. 

I saw the pull request and it seems most of the code has been approved and put in the master branch (?). Does it mean it is available now or will be soon? Thanks.

**Comment by tbodt** [comment]
The code has not been approved by any project committers, and still needs some work before it can be merged. Sorry about that!
